public/app/plugins/datasource/loki/querybuilder/LokiQueryModeller.ts

```diff
@@ -46,7 +57,18 @@ export class LokiQueryModeller extends LokiAndPromQueryModellerBase {
         ],
       },
       {
-        name: 'Log query with parsing and label filter',
+        name: 'Filter log line and parse with json parser',
+        type: LokiQueryPatternType.Log,
+        // {} |= `` | json | __error__=``
+        operations: [
+          { id: LokiOperationId.LineContains, params: [''] },
+          { id: LokiOperationId.Json, params: [] },
+          { id: LokiOperationId.LabelFilterNoErrors, params: [] },
+        ],
+      },
+      {
+        name: 'Parse log line with logfmt parser and use label filter',
```

Comment:
```suggestion
        name: 'Parse log lines with logfmt parser and use label filter',
```

Or `a log line` if needs to be singular.